ui/app/components/customize-gas-modal/index.js

```diff
@@ -244,7 +244,7 @@ CustomizeGasModal.prototype.render = function () {
           min: 1,
           // max: 100000,
           step: 1,
-          onChange: value => this.convertAndSetGasLimit(value),
+          onChange: value => this.convertAndSetGasLimit(String(value)),
```

Comment:
I don't think this change is the. The `conversionUtil` call on line 155 will always return a string.